taichi/ir/statements.h

```diff
@@ -1041,11 +1191,15 @@ class ClearListStmt : public Stmt {
 // Checks if the task represented by |stmt| contains a single ClearListStmt.
 bool is_clear_list_task(const OffloadedStmt *stmt);
 
+/**
+ * TODO: document for InternalFuncStmt
```

Comment:
So... what's an `InternalFuncStmt`?

Reply:
It seems to be used by `ti.call_internal`, which is to run some C++ tests in `taichi/runtime/llvm/internal_functions.h`